Dockerfile

```diff
@@ -30,19 +30,9 @@ ARG NPM_BASE_64_AUTH
 ARG NPM_EMAIL
 ENV DEBIAN_FRONTEND=noninteractive
 
-# install legacy build environment for docs
-RUN apt-get -o Acquire::Check-Valid-Until=false update
-RUN apt-get -y install libpython2.7-dev python2.7 git rsync unzip curl
```

Comment:
Oops, I meant to leave a comment here as well. Everything to the left of `python2.7` we will want to install as well (i.e. we can add  `rsync` and `curl` to the install.